README.md

```diff
@@ -188,6 +188,18 @@ These parameters take precedence over the file on disk if both are specified.
 
 ## Usage
 
+## Accessing Grafana
+
+To access Grafana, use the following in your browser of choice: 
+```http://[AGENT IP ADDRESS]/3000```
+
+When first accessing Grafana, the default login and password are as follows:
+
+Username: Admin
+Password: Admin
+
+Upon first sign in, you will then be prompted to change your password. You have the optiont to change it here, or skip and proceed to the dashboard.
```

Comment:
```suggestion
Upon first sign in, you will then be prompted to change your password. You have the option to change it here, or skip and proceed to the dashboard.
```

Reply:
i also would not suggest changing the password here, it has to be done in the module or alot of the access tokens mess up

Reply:
That's fine, I'll remove that now.